po/de.po

```diff
@@ -20913,6 +20913,11 @@ msgstr "Konvertieren"
 msgid "Missing transaction date."
 msgstr "Buchungsdatum fehlt."
 
+#: gnucash/python/init.py:105
+#, python-brace-format
+msgid "Welcome to {0}!\n"
+msgstr "Willkommen zur {0}!\n"
+
```

Comment:
I wonder if we should use "old style" python formatting, e.g.
```
(_('Welcome to %s') % 'GnuCash')
```
to increase the odds that we can re-use C msgids in Python.

Reply:
> You can do it now for test purposes manually, but in the final version I would prefer, to have 3 separate commits:
> 
>     1. work on python
> 
>     2. merge commits up to xxx into de.po
> 
>     3. update de.po (with msgfmt's statistics as line 3)
I'll do that. I put the change in manually to have some data to test if it is working.

Reply:
@jralls If using old-style formatting leads to a possibly bigger resource pool as translations have been made an can be reused I think that is a good idea. Maybe prefer old style and if not possible (because of complex formatting stuff) use newstyle as fallback.

Reply:
That sounds reasonable.